test/regression/gufi_stats.expected

```diff
@@ -50,6 +51,22 @@ directory/subdirectory 2
 leaf_directory 1
 unusual#? directory , 1
 
+$ gufi_stats --verbose depth "prefix"
+    -B 4096
+    -G SELECT CASE length(name) WHEN 0 THEN 0 ELSE length(name) - length(replace(name, '/', '')) END FROM aggregate ORDER BY name ASC
+    -I CREATE TABLE out(name TEXT)
+    -J INSERT INTO aggregate SELECT * FROM out
+    -K CREATE TABLE aggregate(name TEXT)
+    -S INSERT INTO out SELECT rpath(sname, sroll) FROM vrsummary
+    -d
+    -n 1
+    -y 0
+    -z 0
+    prefix
+   gufi_query
+0
+GUFI query is
```

Comment:
This is flipped

Reply:
oh, maybe I need to do the run_unsorted or whatever. One sec.

Reply:
Fixed and pushed.